README.md

```diff
@@ -4,76 +4,122 @@
+<br >The Red Hat Dependency Analytics extension is an online service hosted and maintained by Red Hat.
+Dependency Analytics only accesses your manifest files to analyze your application dependencies before displaying the vulnerability report.
+
+**IMPORTANT:**
+<br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), and Node ecosystems (`npm`).
+In future releases, Red Hat plans to support other programming languages.
+
+##### Table of Contents
+- [Quick start](#quick-start)
+- [Configuration](#configuration)
+- [Features](#features)
+- [Using Red Hat Dependency Analytics for CI builds](#using-red-hat-dependency-analytics-for-ci-builds)
+- [Know more about the Red Hat Dependency Analytics platform](#know-more-about-the-red-hat-dependency-analytics-platform)
+- [Data and telemetry](#data-and-telemetry)
+- [Support, feedback \& questions](#support-feedback--questions)
+- [License](#license)
+
+## Quick start
+
+**Prerequisites**
+
+- For Maven projects, analyzing a `pom.xml` file, you must have the `mvn` binary in your system’s `PATH` environment.
+- For Node projects, analyzing a `package.json` file, you must have the `npm` binary in your system’s `PATH` environment.
+
+<br >**IMPORTANT:** 
+<br >Visual Studio Code by default executes binaries directly in a terminal found in your system's `PATH` environment.
+You can configure Visual Studio Code to look somewhere else to run the necessary binaries.
+You can configure this by accessing the [extension settings](https://code.visualstudio.com/docs/getstarted/settings).
+Click the **Workspace** tab, search for the word _executable_, and specify the absolute path to the binary file you want to use for Maven or Node.
+
+**Procedure**
+
+1. Install [Visual Studio Code](https://code.visualstudio.com/) on your workstation.
+2. After the installation finishes, open the Visual Studio Code application.
+3. Click the **Extensions** icon from the **Activity Bar** or click **View** from the file menu, and click **Extensions**.
+4. Search the **Marketplace** for _Dependency Analytics_.
+5. Click the **Install** button to install the extension.
+6. To start scanning your application for security vulnerabilities, and view the vulnerability report, you can do one of the following:
+   - Open a manifest file, hoover over a dependency marked by the inline analysis, indicated by the wavy-red line under a version number or dependency name, click **Quick Fix**, and click **Detailed Vulnerability Report**.
+   - Open a manifest file, and click the **pie chart icon** ![ Pie chart icon ](images/0.2.0/icon.png).
+   - Right click on a manifest file in the **Explorer** view, and click **Red Hat Dependency Analytics Report...**.
+7. (OPTIONAL) You can link your Snyk account to Dependency Analytics by doing the following:
+   1. Log into your [Snyk account](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&utm_source=code_ready&code_ready=FF1B53D9-57BE-4613-96D7-1D06066C38C9).
```

Comment:
The link could be simplified: https://app.snyk.io/login/

Reply:
Yes, it could, and I originally had the simplified link, but I learned that because of the partnership between Red Hat and Snyk, we have a custom campaign link that we want to use.